cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -67,13 +72,39 @@
     {% endif %}
 
     {% if prototype %}
```

Comment:
Should I leave or remove? I assume @richaagarwal's PR will get rid of this all at once.